src/ghdcbot/engine/metrics.py

```
@@ -138,6 +138,31 @@ def format_metrics_summary(metrics: UserMetrics | None) -> str:
 
 
+def build_contribution_summary_message(
+    storage: Storage,
+    github_user: str,
+    *,
+    window_days: tuple[int, ...] = (7, 30),
+    rank_subject: str = "you're",
+    now: datetime | None = None,
+) -> str:
+    """Build the multi-window /summary body for a GitHub user (rank among all activity)."""
+    end = now or datetime.now(timezone.utc)
```

Ruff (0.15.21)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)